scripts/schemas/index.js

```diff
@@ -0,0 +1,13 @@
+const communitySchema = require('./community')
+const demosSchema = require('./demos')
+const pluginsSchema = require('./plugins')
+const sitesSchema = require('./sites')
+const startersSchema = require('./starters')
+
+module.exports = {
+  communitySchema,
```

Comment:
nit: this uses two spaces instead of a tab like the other lines.

Reply:
I thought I ran prettier over all the files in scripts …
(I prefer spaces over tabs, but let's not start an argument)

Reply:
I'm on Team "Spaces" as well, but I think Prettier was auto formatting everything w/ tabs in these docs (possibly due to the [.editorconfig file](https://github.com/11ty/11ty-website/blob/master/.editorconfig) or something).